pkgs/development/python-modules/scikit-build/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, wheel, setuptools, packaging }:
+
+buildPythonPackage rec {
+  pname = "scikit-build";
+  version = "0.8.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hh275lj98wgwi53mr9fqk8wh1dajjksch52xjax6a79gld4391a";
+  };
+
+  propagatedBuildInputs = [ wheel setuptools packaging ];
+  
+  # Unable to find pytest-runner, so disabled.
+  doCheck = false;
+
+  meta = {
+    homepage = http://scikit-build.org/;
+    description = "Improved build system generator for CPython C/C++/Fortran/Cython extensions.";
+    license = lib.licenses.mit;
```

```suggestion
    license = with lib.licenses; [ mit bsd2 ];
```
See https://github.com/scikit-build/scikit-build/blob/master/LICENSE. Also add a comment explaining why there are two.